portfolio/src/main/webapp/index.html

```diff
@@ -97,7 +97,59 @@ <h2>Projects</h2>
           </li>
         </ul>
 
-        <h2>Fun</h2>
+        <h3>Contact Me</h3>
+        <form action="/form-handler" method="POST" id="message-form">
+          <div class="mb-3">
+            <label for="email" class="form-label">Email address</label>
+            <input
+              type="email"
+              class="form-control"
+              name="email"
+              id="email"
+              placeholder="name@example.com"
+              required
+            />
+          </div>
+          <div class="mb-3">
+            <label for="message" class="form-label"
+              >Leave a message for me</label
+            >
+            <textarea
+              class="form-control"
+              name="message"
+              id="message"
+              rows="3"
+              placeholder="your message"
+              required
+            ></textarea>
```

Comment:
```suggestion
            <label for="message" class="form-label">
                Leave a message for me
            </label>
            <textarea
              class="form-control"
              name="message"
              id="message"
              rows="3"
              placeholder="your message"
              required></textarea>
```

It's nice to keep the closing brackets together. This is probably more optional than strictly required, but helps with readability. Please apply this to all instances here and below.

Reply:
Hi Emily! I believe this is due to auto-formatting from using prettier as a pre-commit hook. Would you recommend getting rid of the hook? Or is there a better work-around?

Reply:
Hmm, is there a way to configure prettier somehow? It seems like it has configs but not sure if you have a way to use them: https://prettier.io/docs/en/configuration.html

if not, I guess it's not a huge deal, but just something to be aware of.

Reply:
I played around with prettier config a bit, but did not work out. I ended up getting rid of prettier pre-commit hook if that's ok (will use the editor's default formatting tool going forward). Will merge this PR.